### PR TITLE
feat(replay): Implement mouse tracking along the scrubber and timeline

### LINKED ISSUE #33626: When hovering over the player scrubber, highlight the same time position in the span viewer
When hovering over a position in the scrubber:
image

We want to highlight that same timestamp on this span view header:

image

(This component above is named `<ActualMinimap>`)

It might be easier to start this off in storybook and we should probably discuss the approach before implementation (e.g. should we try to extend `<ActualMinimap>` or copy and paste for now).

### PR DESCRIPTION
**Pull request by ryan953**
This adds mouse tracking to the video `<Scrubber>` as well as the `<ReplayBreadcrumbOverview>` timeline component.

When the mouse is inside one of those components then `currentHoverTime` will be set and an additional scrubber bar will appear and will follow the mouse position.

**First Example**
- the video is paused at `0:58`, as indicated by the dark purple bar.
- the mouse is near `1:27` and the transparent-purple bar is extended past the dark purple bar to track it (mouse isn't captured in the screen shot)

Screen Shot 2022-05-09 at 6 53 27 PM

**Second example**
- Video paused at the same spot
- Mouse is at the `0:30`, so the mouse tracking indicators are sticking out around the dark-purple bars.

Screen Shot 2022-05-09 at 6 53 33 PM

Fixes #33626

### CODE REVIEW

**Comment by ryan953** [comment]
Current dependencies on/for this PR:
* master
  * **PR #34359** <a href="https://app.graphite.dev/github/pr/getsentry/sentry/34359" target="_blank">Graphite</a>  👈
    * **PR #34360** <a href="https://app.graphite.dev/github/pr/getsentry/sentry/34360" target="_blank">Graphite</a> 
      * **PR #34361** <a href="https://app.graphite.dev/github/pr/getsentry/sentry/34361" target="_blank">Graphite</a> 
        * **PR #34362** <a href="https://app.graphite.dev/github/pr/getsentry/sentry/34362" target="_blank">Graphite</a> 
          * **PR #34363** <a href="https://app.graphite.dev/github/pr/getsentry/sentry/34363" target="_blank">Graphite</a> 
            * **PR #34364** <a href="https://app.graphite.dev/github/pr/getsentry/sentry/34364" target="_blank">Graphite</a> 
              * **PR #34365** <a href="https://app.graphite.dev/github/pr/getsentry/sentry/34365" target="_blank">Graphite</a> 
                * **PR #34366** <a href="https://app.graphite.dev/github/pr/getsentry/sentry/34366" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/getsentry/sentry/34359?utm_source=stack-comment).

**Review by billyvg — APPROVED** [review]

**Review by SDaian — APPROVED** [review]

**Comment by ryan953** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/getsentry/sentry/34359?utm_source=stack-comment).


Job Reference: KCvw6i8GL2q9gcyafKWF

**Pull request merged**
